### ISSUE TITLE
Reproduce the results from the paper

### ISSUE THREAD

**Issue by J4nn4** [ZixuanKe/PyContinual#12]
Hello, 
thank you very much for the framework implementation.
I would like to reproduce the results from the paper "Achieving Forgetting Prevention and Knowledge Transfer in Continual Learning".
Could you please provide the configuration file with parameters of the experiments? With what parameters should I run the code to be able to reproduce the results.
Many thanks in advance!

**Comment by ZixuanKe** [comment]
Thank you so much for your interest. Please see "https://github.com/ZixuanKe/PyContinual/blob/main/docs/run_exist.md"

More specifically, you can run 

```    
python  run.py \  
		--bert_model 'bert-base-uncased' \   
		--backbone bert_adapter \  
		--baseline ctr \  
		--task asc \  
		--eval_batch_size 128 \  
		--train_batch_size 32 \  
		--scenario til_classification \  
		--idrandom 0  \
		--use_predefine_args
```
``use_predefine_args`` will load the predefined hyper-parameters for you.

Feel free to let us know if you have further questions.

**Comment by J4nn4** [comment]
Thank you so much for your provided answer.

Since my last questions I have done some extensive testing with your framework, but I am still not able to achieve the improvement over standalone baselines reported in the paper.

I've made some additional tests and even if I train the same task (aspect sentiment classification) with multiple capsules on the same dataset (XuSemEval14_laptop), the capsules do not support each other and even show opposite behavior, achieving lower accuracies and giving results like this:

Accuracies =
	 75.2%   0.0% 
	 68.2%  67.7% 

Have there been any major updates in the repository that I should consider that could be causing this model behavior? I would appreciate your suggestions in this regard.

**Comment by ZixuanKe** [comment]
Thank you for your feedback. To verify the results, I ran the CTR using the following script
```
 python  run.py \
  --bert_model 'bert-base-uncased' \
  --backbone bert_adapter \
  --baseline ctr\
  --task asc \
  --eval_batch_size 128 \
  --train_batch_size 32 \
  --scenario til_classification \
  --idrandom 0  \
  --use_predefine_args
```
I got the following progressive results (accuracy)

```0.7539	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000
0.7429	0.8232	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000
0.7414	0.8027	0.9259	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000
0.7445	0.7866	0.8889	1.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000
0.7257	0.7884	0.9259	1.0000	0.8000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000
0.7116	0.7857	0.9259	1.0000	0.8500	0.8372	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000
0.7100	0.7786	0.9259	0.9545	0.9000	0.7674	0.8913	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000
0.7304	0.7875	0.9259	1.0000	0.9000	0.8140	0.9348	0.9167	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000
0.7351	0.7911	0.9259	1.0000	0.9000	0.8372	0.9565	0.9167	1.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000
0.7367	0.7902	0.9259	1.0000	0.9000	0.8605	0.9348	0.9167	1.0000	0.9412	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000
0.7414	0.7830	0.9630	1.0000	0.9000	0.8140	0.9348	0.9167	1.0000	0.9294	0.8400	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000
0.7226	0.7741	0.9630	1.0000	0.8500	0.8372	0.9348	0.9167	1.0000	0.9294	0.8000	0.9180	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000
0.7241	0.7777	0.9630	1.0000	0.9000	0.8372	0.9348	0.9167	1.0000	0.9294	0.8000	0.9016	0.9773	0.0000	0.0000	0.0000	0.0000	0.0000	0.0000
0.7257	0.7786	0.9630	1.0000	0.9000	0.8605	0.9130	0.9167	1.0000	0.9059	0.8400	0.9180	0.9773	0.8261	0.0000	0.0000	0.0000	0.0000	0.0000
0.7194	0.7777	0.9630	1.0000	0.9000	0.8372	0.9130	0.9167	1.0000	0.9059	0.8800	0.9344	0.9773	0.8696	0.9412	0.0000	0.0000	0.0000	0.0000
0.7100	0.7571	0.9630	1.0000	0.8500	0.8140	0.9130	0.9583	1.0000	0.8824	0.8800	0.9180	0.9545	0.8696	0.9706	1.0000	0.0000	0.0000	0.0000
0.7241	0.7839	0.9630	1.0000	0.9500	0.7907	0.9130	0.9583	1.0000	0.8941	0.9200	0.9344	0.9773	0.8696	0.9412	1.0000	0.8966	0.0000	0.0000
0.7163	0.7893	0.9630	1.0000	0.9500	0.7442	0.9130	0.9167	1.0000	0.9294	0.8800	0.9344	0.9773	0.8696	0.9412	1.0000	0.8966	0.8333	0.0000
0.7116	0.7795	0.9630	1.0000	1.0000	0.7674	0.9348	0.9167	1.0000	0.9176	0.9200	0.9344	0.9545	0.8261	0.9706	1.0000	0.8966	0.8889	0.7097
```
The average accuracy after continual learning (i.e., the average over the numbers in the last row) is 0.8995, which is very similar to the reported table (0.8947).
